### ISSUE TITLE
RunProject with 2 arguments is not properly working

### ISSUE THREAD

**Issue by romcab** [christianrondeau/SikuliSharp#7]
Hi there,

Thank you for creating this wrapper.

I have a sikuli script that automates installation of firefox. It accepts command line argument - filename. In my c# code, I tried to call it with Sikuli.RunProject(args[0], args[1]) where 
args[0] = path of firefox.sikuli
args[1] = "--args firefox-installer-here"

Upon running, i noticed that app focus remains the "cmd". I tried to edit my script to hardcode the filename and just call the Sikuli.RunProject(args[0]) and it runs successfully. Any recommendation?

**Comment by christianrondeau** [comment]
The arguments are sent as-is to sikuli, following the project name, so if Sikuli accepts them, that should work (see https://github.com/christianrondeau/SikuliSharp/blob/master/SikuliSharp/Sikuli.cs#L34)

Try running `sikuli -r your-path --args firefox-installer-here` and see if it works?

**Comment by romcab** [comment]
Thanks for your response, I appreciate it!

I just noticed that if run the command in cmd.prompt, it fails because App.focus is in cmd.exe. However, if I wrapped it in a batch file, it works perfectly. I might ask or checked the launchpad to see explanation.

runsikulix,cmd -r project-path --args firefox-installer-here

**Comment by christianrondeau** [comment]
Just to be sure; you are saying that it works using the Sikuli command line directly, or not? Also, can you share the exact string you are using? I won't be able to try it from here, but maybe that'll spark some ideas.

**Comment by romcab** [comment]
Nope, it did not work. below is my script and the command i invoke in the command prompt. absetup.exe is avbrowser installer. basically, i want to automate its installation.

runsikulix.cmd -r my-script.sikuli --args 'absetup.exe'

my-script.sikuli:
import sys
#name = "absetup.exe"
name = sys.argv[1]
print(name)

myApp = App(name)

# wait to open
if not myApp.window():
    myApp.open(5)
    s.wait(5)

# shift the focus to window
myApp.focus()
wait(5)
click("Install.png")
s.wait(10)
click("CheckBox.png")
s.wait(1)
click("Finish.png")
